src/SALib/sample/saltelli.py

```diff
@@ -1,34 +1,79 @@
 from __future__ import division
+import warnings
 
+import math
 import numpy as np
 
 from . import common_args
 from . import sobol_sequence
 from ..util import scale_samples, nonuniform_scale_samples, read_param_file, compute_groups_matrix
 
 
-def sample(problem, N, calc_second_order=True, seed=None, skip_values=1000):
-    """Generates model inputs using Saltelli's extension of the Sobol sequence.
+def sample(problem, N, calc_second_order=True, seed=None, skip_values=1024):

```

Comment:
The seeding could also be removed. But might break some codes.
```suggestion
def sample(problem, N, calc_second_order=True, skip_values=1024):

```

Reply:
I'll keep `seed` for now, but might remove in the dev branch